test/biophysics.jl

```diff
@@ -89,26 +96,29 @@ Q_in = Q_solar + Q_IR_in + Q_metab
 Q_out = Q_IR_out + Q_conv + Q_evap + Q_resp + Q_cond
 Q_in - Q_out
 
-T_x = T_air
 
 T_surf_s = find_zero(heat_balance, (T_air - 40K, T_air + 100K), Bisection())
 T_surf_C = (Unitful.ustrip(T_surf_s) - 273.15)°C
 
 
 # using structs to pass parameters
 
+# define the geometry
 mass_organism = 0.04kg
 ρ_organism = 1000kg/m^3
 shapeb_organism = 2
 shape_organism = Cylinder(mass_organism, ρ_organism, shapeb_organism)
 body_organism = Body(shape_organism, Naked())
 
+# construct the Model which holds the parameters of the organism in the Organism concrete struct, of type AbstractOrganism
 lizard = Model(Organism(body_organism, OrganismParams()))
+# get the environmental parameters
 environmental_params = EnvironmentalParams()
+# get the variables for both the organism and environment
 variables = (organism=OrganismalVars(), environment=EnvironmentalVars())
```

Comment:
this line is throwing the error: BoundsError: attempt to access NTuple{13, Float64} at index [14]

Reply:
I don't get that error. You are using `Model` so you have run `using ModelParameters`. Maybe the version of the package? I am using [4744a3fa] ModelParameters v0.3.7 in julia Version 1.8.5 (2023-01-08).

Reply:
Ah I'm using ModelParameters v0.1.0 - having trouble working out how to upgrade it because 'up ModelParameters' not doing the trick.

Reply:
just tracked that issue down to me having DynamicEnergyBudgets installed which is limited to using v0.1.0